src/hooks/useReactToPrint.ts

```diff
@@ -1,6 +1,6 @@
 import { useCallback } from "react";
 
-import { Font } from "../types/Font";
+import { Font } from "../types/font";
```

Comment:
Is this change intentional?

Reply:
Can you resolve this if lower case works fine for you as well? Do you work with NTFS? For case sensitive FS, this should lead to an error.